deploy/jN/lib/es5-shim.js

```diff
@@ -0,0 +1,29 @@
+/**!
```

Comment:
Is there not minified version?

Reply:
The link is present in the beginning of each file. The problem is that

1. When I added the file and the link, the primary purpose of adding the link of that was to give credit to the lib authors and to provide the link where I can get the lib upgrade.
2.  CommonJS wrapper was removed. The lib names were either externed or exported to prevent their renamings and being optimized out.
3. The stuff was then processed with Google Closure Compiler (a JS optimizer, not just in the sense of minification) on ADVANCED level. Then the code was formated using JSFormat plugin, to make the editing easier.
4. Then I have done some manual fixes, since Google Closure Compiler on advanced level may spoil the code.

 Each step was checked using dynamic code execution in CurrentView.

 All of these were done long long long ago (I don't remember how long exactly, the time in git commit is fake, I have splitted a large commit into multiple ones, and that large commit had time of its last modification), probably we may need to upgrade the lib.

So there is not really a "non-minified" exist, I have not kept it.

I have given you the general recepie what should be done in order to upgrade the lib. Or maybe we need some kind of script doing this automatically.